.harness/base_checks.yaml

```diff
@@ -40,6 +40,7 @@ pipeline:
                             - "3.10"
                             - "3.11"
                             - "3.12"
+                            - "3.13"
```

Comment:
I think you will also need to update constraint in drum to run this test right?

Reply:
oh yeah